### DISCUSSION TITLE
Process of voting for issues

### DISCUSSION THREAD

**Discussion by AlmasB** [gluonhq/scenebuilder#488] [Ideas]
Just leaving this here as the number of issues is growing and we still need to go through the backlog. Given limited resources, in the long run, we need to ensure that work is primarily focused on issues that are of high priority to users. Therefore, we need a way for users to flag issues they wish to be fixed sooner.

Given emojis are quite easy to leave on an issue, we could simply pick an emoji (propose ❤️ since 👍  is commonly used and may mean the user supports the issue but not necessarily wishes for it to be prioritised) that will represent the number of votes. The emoji should be used on the first post (description) of the issue for easier calculation.

Taking the idea further, we can have a "high priority" label, which is added once ❤️ reaches a certain number. It would be ideal that each new major release, e.g. 19, 20, etc. should contain (at least considerations of, leaving us some leeway here) fixes for one "high priority" issue.

**Comment by eugener** [comment]
Another way of accomplishing this is to use projects with customized board, i.e. custom column(s) for high priority issues.

**Comment by Oliver-Loeffler** [comment]
Very good idea Almas. I also like the idea with the customized boards as @eugener mentioned.
